traktorml.py

```diff
@@ -0,0 +1,196 @@
+from collections import defaultdict
+from typing import List, Tuple, Union, Dict
+
+import mlflow
+from pandas import DataFrame
+from pydantic import BaseModel
+from textual.containers import ScrollableContainer
+from textual.events import Key
+from textual.widgets import DataTable, Footer
+from textual import on
+from textual.app import App, ComposeResult
+from textual.widgets import Header, Select
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
```

Comment:
You don't need both. `load_dotenv()` is enough

Reply:
Good point.